xbmc/platform/darwin/osx/peripherals/PeripheralBusUSB.cpp

```diff
@@ -237,10 +237,9 @@ void CPeripheralBusUSB::DeviceAttachCallback(CPeripheralBusUSB* refCon, io_itera
           }
         }
         if (!ttlDeviceFilePath.empty())
-          privateDataRef->result.m_strLocation =
-              StringUtils::Format("{}", ttlDeviceFilePath.c_str());
+          privateDataRef->result.m_strLocation = std::to_string(ttlDeviceFilePath);
```

Comment:
Same as the other one, ttlDeviceFilePath is already a string, drop the to_string()